src/aggregate/aggregate_exec.c

```diff
@@ -154,6 +154,11 @@ static int sendChunk(AREQ *req, RedisModuleCtx *outctx, size_t limit) {
   int rc = RS_RESULT_EOF;
   ResultProcessor *rp = req->qiter.endProc;
 
+  if (!(req->reqflags & QEXEC_F_IS_CURSOR) && 
+      !(req->reqflags & QEXEC_F_IS_SEARCH)) {
+    limit = RSGlobalConfig.maxAggregateResults;
```

Comment:
This will be breaking change no? Cause maxAggregateResults is set to 1M be the default.